communitiesuk/funding-service-design-frontend
# FS-2037: Welsh translation added for submitted application page

https://mhclgdigital.atlassian.net/browse/FS-2037

### Change description
- Welsh translation added for submitted application page



### How to test
- Select Welsh language, complete it and verify that the submit_application page has the correct Welsh translation


### Screenshots of UI changes (if applicable)
